mllib/src/test/java/org/apache/spark/ml/feature/JavaPCASuite.java

```diff
@@ -107,7 +107,11 @@ public VectorPair call(Tuple2<Vector, Vector> pair) {
       .fit(df);
     List<Row> result = pca.transform(df).select("pca_features", "expected").toJavaRDD().collect();
     for (Row r : result) {
-      Assert.assertEquals(r.get(1), r.get(0));
+      Vector calculatedVector = (Vector)r.get(0);
```

Comment:
Fix the cast issue, although the style checker does not complain anything. I think we can track the style checker issue in a separate JIRA.